src/aes.rs

```diff
@@ -17,7 +17,7 @@ bytes!(RCon, 11);
 bytes!(Bytes144, 144);
 bytes!(Bytes176, 176);
 
-const SBOX: SBox = SBox([
+const SBOX: SBox = SBox(secret_bytes!([
```

Comment:
It would be nicer if we wouldn't have to call the `secret_bytes!` here. How about adding a function to `SBox` (so arrays in general) to make them secret?

Reply:
Indeed it would be nicer, but then it wouldn't be `const` anymore. In `const` declarations, you can only use constructors and literal values; no functions allowed. The `secret_bytes!` macro is a trick to avoid writing the `U8` constructor for each array value.

Reply:
Good point... Let's keep it like this for now and see later if we can make it nicer.